src/battle_game/core/bullets/split_bullet.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include "battle_game/core/bullet.h"
+
+namespace battle_game::bullet {
+class SplitBullet : public Bullet {
+ public:
+  SplitBullet(GameCore *core,
+              uint32_t id,
+              uint32_t unit_id,
+              uint32_t player_id,
+              glm::vec2 position,
+              float rotation,
+              float damage_scale,
+              glm::vec2 velocity);
+  ~SplitBullet() override;
+  void Render() override;
+  void Update() override;
+  void ResetCountDown();
+
+ private:
+  glm::vec2 velocity_{};
+  uint32_t split_count_down_{0};
+  uint32_t split_interval = 15;
```

Comment:
```suggestion
  uint32_t split_interval_ = 15;
```

For private variables: lower_case_ ending with an underscore